docs/guides/migrating-from-camunda-7/data-migrator/limitations.md

```diff
@@ -112,6 +112,14 @@ To migrate a subprocess that is started from a call activity, the migrator must
 
 Processes with active multi-instance elements can currently not be migrated. We recommend to finish the execution of any multi-instance elements prior to migration.
 
+#### Parallel gateways
+
+Process instances with active joining parallel gateways cannot currently be migrated. The migrator will skip these instances during migration.
+
+- This limitation occurs when some execution paths have completed and reached the joining parallel gateway, but other paths are still waiting at activities before the gateway.
+- Recommendation: Allow all parallel execution paths to complete and tokens to move past the joining gateway before initiating the migration.
+- See https://github.com/camunda/camunda-bpm-platform/issues/5461 for the follow-up work to resolve this limitation.
```

Comment:
```suggestion
- See https://github.com/camunda/camunda-bpm-platform/issues/5461
```

"follow-up work to resolve this limitation" might sound, from a user perspective, like we are working on it. The truth is, we will work on it based on the feedback or the complexity of the resolution.